src/clj/kixi/hecuba/data/measurements/calculations.clj

```diff
@@ -434,3 +437,54 @@
           tariffs                       (read-tariffs store entity_id)]
       (doseq [timestamp (time/seq-dates start-date end-date (t/days 1))]
         (expenditure-hourly store sensor timestamp (t/plus timestamp (t/days 1)) tariffs)))))
+
+(defn sum-up-expenditure
+  "Calculate total and insert it into the database. Update output sensor's dirty dates."
+  [store calculated-sensor start end measurements]
+  (db/with-session [session (:hecuba-session store)]
+    (let [{:keys [sensor_id device_id]} calculated-sensor
+          calculated-data  [{:value (str (apply + (map :value (filter #(number? (:value %)) measurements))))
```

Comment:
Is this always going to be a small enough sequence that doing the apply on the mapped filtered values won't blow things up or should it be (reduce + ...) ?